docs/references/faq.md

```diff
@@ -0,0 +1,137 @@
+# FAQ 
+
+## Where are the debug logs located?
+
+Internal tool logs are written to a file called PythonSDK.log located in Local AppData. For example, `%LOCALAPPDATA%\Alteryx\Log\PythonSDK.log`. The logger object within a tool also writes to this file.
+
+:information_source: `%LOCALAPPDATA%` usually stands for `C:\users\{username}\AppData\Local\`. For more information, refer to `FOLDERID_LocalAppData` [here](https://learn.microsoft.com/en-us/windows/win32/shell/knownfolderid).
+
+## How to get DEBUG level logs?
+
+Ensure you launch the plugin with the environment variable `AYX_SDK_VERBOSE=1`.
+For example, one way to do this is in the Windows Environment dialog. Windows 10+ users, go to [How Do I Set System Environment Variables in Windows 10](https://superuser.com/questions/949560/how-do-i-set-system-environment-variables-in-windows-10_E) for more information.
+
+## _"This tool is only supported in the AMP Engine"_
+
+If you encounter this error (as shown below) _"This tool is only supported in the AMP Engine. Please enable AMP to use this tool."_:
+
+![Error (use AMP)](assets/tool_is_only_supported_in_amp.png)
+ 
+Check that the AMP engine is enabled. In Designer's Workflow Configuration window, select the Runtime tab and ensure the Use AMP Engine check box is checked.
+
+![Toggle AMP](assets/use_amp.png)
+ 
+If you still encounter the error with the correct configuration, make sure you are using a plugin produced with the SDK V2 and up.
```

Comment:
@jsong785 Update to:

If you still encounter the error with the correct configuration, make sure the plugin was produced with version 2+ of the SDK.

Reply:
Handled in https://github.com/alteryx/ayx-developer-sdk/pull/6